LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/constants/B2CProvider.java

```diff
@@ -41,4 +43,8 @@ public enum B2CProvider {
     public String toString() {
         return value;
     }
+
+    public static B2CProvider fromName(@NonNull final String name) {
```

Comment:
Why is this method necessary? Can we refactor the code that's using to just use the enum directly?

Reply:
I'll still need this method to convert from B2CProvider in interaction/B2CProvider when creating a labquery and useing the providerName to get a B2CProvider. AbstractB2CTest in MSAL.